### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by yv-official**
added fields User model

### CODE REVIEW

File: server/helpers.py [hunk 1]
```diff
@@ -9,16 +9,22 @@ def register_parser():
...
     loginparser = reqparse.RequestParser()
-    loginparser.add_argument('email', help='This Field Cannot be blank', required=True)
+    loginparser.add_argument('username', help='This Field Cannot be blank', required=True)
     loginparser.add_argument('password', help='This Field Cannot be blank', required=True)
     return loginparser
 
 # helper for userRegistration
-def userRegister(username, email, password):
+def userRegister(username, email, password, firstName, lastName, gender, phone, branch, year):
     check_username = users.User.objects(username=username).first()
```
Comment (Logan-47) [inline server/helpers.py:28]: Check username function is created but never used
it should  be checked wthether a user with the same username exists or not beacuse we are using username to login the user

If user with the same username exists Raise an exception username already exists.

File: server/helpers.py [hunk 2]
```diff
@@ -9,16 +9,22 @@ def register_parser():
     registerparser.add_argument('username', help = 'This Field Cannot be blank', required=True)
     registerparser.add_argument('email', help = 'This Field Cannot be blank', required=True)
     registerparser.add_argument('password', help = 'This Field Cannot be blank', required=True)
+    registerparser.add_argument('firstName', help = 'This Field Cannot be blank', required=True)
+    registerparser.add_argument('lastName')
+    registerparser.add_argument('gender')
```
Comment (Logan-47) [inline server/helpers.py]: This should be required

File: server/helpers.py [hunk 3]
```diff
@@ -9,16 +9,22 @@ def register_parser():
...
+    registerparser.add_argument('phone')
+    registerparser.add_argument('branch')
```
Comment (Logan-47) [inline server/helpers.py]: this should be required

File: server/responses.py [hunk 4]
```diff
@@ -17,8 +18,10 @@ def userRegistrationSuccessful(user, access_token, refresh_token):
 # response for Successful login
 def userSuccessfulLogin(user, access_token, refresh_token):
     res = make_response(jsonify({
+            'userId': str(user.id),
             'username': user.username,
-            'message': f"Logged in as {user.email}",
+            'name': user.firstName + ' ' + user.lastName,
+            'message': f"Logged in as {user.username}",
```
Comment (Logan-47) [inline server/responses.py]: lets change 'message' to 'msg'

**Review by Logan-47 — CHANGES_REQUESTED** [review]

File: server/helpers.py (hunk 1 above)
Comment (yv-official) [reply server/helpers.py:28]: ok

Comment (yv-official) [reply server/helpers.py:28]: check_username function is in use. it is not displayed here because GitHub is not comparing it with change

Comment (Logan-47) [reply server/helpers.py:28]: ok

**Review by Logan-47 — APPROVED** [review]

**Pull request merged**